src/Renderer.ts

```diff
@@ -388,16 +442,16 @@ export default class Renderer {
     return m;
   }
 
-  _renderSkin(
-    skin,
-    drawMode,
-    matrix,
-    scale,
-    effects,
-    effectMask,
-    colorMask,
-    spriteColorId
-  ) {
+  private _renderSkin(
+    skin: Skin,
+    drawMode: DrawMode,
+    matrix: MatrixType,
+    scale: number,
+    effects?: _EffectMap,
+    effectMask?: number,
+    colorMask?: [number, number, number, number],
+    spriteColorId?: number
```

Comment:
I feel like those can provide a false sense of security since TypeScript doesn't actually support nominal typing. I'll create `RGBA`/`RGBANormalized` type aliases for the color tuples anyway.